Java/IterativeFibonacci.java

```diff
@@ -0,0 +1,32 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class IterativeFibonacci{
+	static int range;
+	public static void main(String[] args) throws NumberFormatException, IOException {
+		
+		System.out.println("Enter range to print Fibonacci Series : ");
+		BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+		range = Integer.parseInt(reader.readLine());
+		
+		//Call to display FIbonacci series iteratively
+		iterativeFibonacci(range);
+	}
+	
+	public static void iterativeFibonacci(int range){
+		int a=0,b=1,sum=0;
+		System.out.print(0+","+1+",");
+		while(true){
```

Comment:
As a best practice, we could avoid infinite loop here and use while((sum+b)<range) here I think. Let me know your thoughts.